examples/randomWalk.js

```diff
@@ -11,13 +11,33 @@ function stepFunc() {
     return pg.geometry.toCircle(2 * Math.PI * Math.random())
 }
 
+function evenCircleStart(numPoints, radius){
+    return pg.geometry.circlePoints(numPoints, radius);
+}
+
+function archSpiralStart(a, b, n, pointsPerCycle){
+    var startPoints = Array(n);
+    var theta = 0;
+    for(var i = 0; i < n*pointsPerCycle; i++){
+        theta += 2*Math.PI/pointsPerCycle;
```

Comment:
Mutating theta over time seems like it could lead to confusing code.
var theta = 2 * Math.PI * i / pointsPerCycle